java-checks/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json

```diff
@@ -380,6 +380,7 @@
     "S5411",
     "S5413",
     "S5443",
+    "S5445",
```

Comment:
I would expect to see S2976 removed from the SonarWay profile, is it expected to keep it?

Reply:
I asked Eric, he said, it should be done automatically, once we deprecate it.
I don't understand exactly what is the process here. rule-api, didn't remove it, when I run scripts.

I went through this[ doc ](https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/665976/Coding+Rules+Guidelines) and it looks like we need to update RSPEC itself and remove it from SonarWay, but I am not absolutely sure what's done with script and what should be done manualy.

We can also wait till tomorrow to see if any jobs did some updates on the rule.

Reply:
Indeed, "sonartech" did some changes to the RSPEC this night, removing the default profile.

Reply:
Updated PR :)